src/network/netdev/bareudp.c

```diff
@@ -0,0 +1,139 @@
+/* SPDX-License-Identifier: LGPL-2.1+
+ * Copyright © 2020 VMware, Inc. */
```

Comment:
are the explicitly Copyright lines required? We tried hard to remove them from our tree, would suck having to add them here again.

Reply:
hmm yes